libs/sql-parser/src/main/java/io/crate/sql/SqlFormatter.java

```diff
@@ -679,6 +680,12 @@ protected Void visitLongLiteral(LongLiteral node, Integer indent) {
             return null;
         }
 
+        @Override
+        protected Void visitIntegerLiteral(IntegerLiteral node, Integer indent) {
+            builder.append(String.format(Locale.ENGLISH, "%d", node.getValue()));
```

Comment:
```suggestion
            builder.append(Integer.toString(node.getValue()));
```

Cheaper than going through `String.format`

Reply:
Afaik it could be just `builder.append(node.getValue())` as the `StringBuilder` is already doing similar.